game/scripts/npc/abilities/huskar_berserkers_blood.txt

```diff
@@ -29,6 +29,12 @@
         "maximum_health_regen"                            "20 40 60 80 100 120"
         "LinkedSpecialBonus"                              "special_bonus_unique_huskar_6"
       }
+      "04"
```

Comment:
Valve did this. I don't like it

Reply:
leave the magic resistance talent as is, if it's OP we can either remove one or other.

Reply:
With the talent he can reach 99% magic resistance very easy. Please stop talking nonsense.